test/integration/crdvalidation.go

```diff
@@ -1,17 +1,19 @@
 package integration
 
 import (
-	"github.com/onsi/ginkgo"
-	"github.com/onsi/gomega"
 	"github.com/RHsyseng/operator-utils/pkg/validation"
 	"github.com/ghodss/yaml"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
 	brokerv2alpha1 "github.com/rh-messaging/activemq-artemis-operator/pkg/apis/broker/v2alpha1"
+	brokerv2alpha2 "github.com/rh-messaging/activemq-artemis-operator/pkg/apis/broker/v2alpha2"
```

Comment:
do we need to keep both versions imported? or just v2alpha2?

Reply:
we need both versions, as there are more than once CRD files and other CRD version is v2alpha1